frontend/src/views/auth/ResetPassword.vue

```diff
@@ -56,9 +56,8 @@
           v-model="confirmation"
           :label="$tc('views.auth.resetPassword.passwordConfirmation')"
           name="Confirmation"
-          vee-id="confirmation"
-          vee-rules="pwConfirmed:@password"
-          vee-rules-old="confirmed:password"
+          vee-rules="required"
+          :rules="pw2Rules"
```

Comment:
I'd tend to avoid mixing vuetify validation and vee-validate, unless absolutely necessary.

Reply:
Agreed, in the meantime I was able to return to native vee-validate only.